duqtools/status.py

```diff
@@ -180,11 +180,12 @@ def detailed_status(self):
 
 
 def status(progress: bool, detailed: bool, **kwargs):
-    tracker = Status()
-
-    if detailed:
-        tracker.detailed_status()
-    elif progress:
-        tracker.progress_status()
-    else:
-        tracker.simple_status()
```

Comment:
I think you overwrote your own previous adjustements here? I think the logic is still needed right?